### ISSUE TITLE
Incompatible with PAL/Flan

### ISSUE THREAD

**Issue by Alkiasan** [John-Paul-R/Essential-Commands#66]
Whenever changing gamemode with PlayerAbilityLib/Flan installed (Seems to be when flying), this error is posted in console:
```
[19:41:25] [Server thread/WARN]: Player ability minecraft:mayfly was updated externally (expected disabled, was enabled).
java.lang.RuntimeException: stacktrace
        at io.github.ladysnake.pal.impl.VanillaAbilityTracker.checkConflict(VanillaAbilityTracker.java:72) ~[pal-1.3.0.jar:?]
        at net.minecraft.class_3222.handler$zjp000$checkAbilityConsistency(class_3222.java:3261) ~[intermediary-server.jar:?]
        at net.minecraft.class_3222.method_7355(class_3222.java:1256) ~[intermediary-server.jar:?]
        at com.fibermc.essentialcommands.PlayerData.updateFlight(PlayerData.java:237) ~[essential_commands-0.16.1-mc1.17.jar:?]
        at net.minecraft.class_3222.handler$bbk000$onChangeGameMode(class_3222.java:5229) ~[intermediary-server.jar:?]
        at net.minecraft.class_3222.method_7336(class_3222.java:1279) ~[intermediary-server.jar:?]
        at net.minecraft.class_3064.method_13387(class_3064.java:59) ~[intermediary-server.jar:?]
        at net.minecraft.class_3064.method_13391(class_3064.java:32) ~[intermediary-server.jar:?]
        at com.mojang.brigadier.CommandDispatcher.execute(CommandDispatcher.java:263) ~[intermediary-server.jar:?]
        at com.mojang.brigadier.CommandDispatcher.execute(CommandDispatcher.java:177) ~[intermediary-server.jar:?]
        at net.minecraft.class_2170.method_9249(class_2170.java:251) ~[intermediary-server.jar:?]
        at net.minecraft.class_3244.method_14370(class_3244.java:1222) ~[intermediary-server.jar:?]
        at net.minecraft.class_3244.method_31286(class_3244.java:1206) ~[intermediary-server.jar:?]
        at net.minecraft.class_3244.method_12048(class_3244.java:1189) ~[intermediary-server.jar:?]
        at net.minecraft.class_2797.method_12115(class_2797.java:30) ~[intermediary-server.jar:?]
        at net.minecraft.class_2797.method_11054(class_2797.java:7) ~[intermediary-server.jar:?]
        at net.minecraft.class_2600.method_11072(class_2600.java:21) ~[intermediary-server.jar:?]
        at net.minecraft.class_3738.run(class_3738.java:18) ~[intermediary-server.jar:?]
        at net.minecraft.class_1255.method_18859(class_1255.java:151) ~[intermediary-server.jar:?]
        at net.minecraft.class_4093.method_18859(class_4093.java:23) ~[intermediary-server.jar:?]
        at net.minecraft.server.MinecraftServer.method_24306(MinecraftServer.java:788) ~[intermediary-server.jar:?]
        at net.minecraft.server.MinecraftServer.method_18859(MinecraftServer.java:164) ~[intermediary-server.jar:?]
        at net.minecraft.class_1255.method_16075(class_1255.java:125) ~[intermediary-server.jar:?]
        at net.minecraft.server.MinecraftServer.method_20415(MinecraftServer.java:770) ~[intermediary-server.jar:?]
        at net.minecraft.server.MinecraftServer.method_16075(MinecraftServer.java:764) ~[intermediary-server.jar:?]
        at net.minecraft.class_1255.method_18857(class_1255.java:134) ~[intermediary-server.jar:?]
        at net.minecraft.server.MinecraftServer.method_16208(MinecraftServer.java:749) ~[intermediary-server.jar:?]
        at net.minecraft.server.MinecraftServer.handler$cfh000$modifiedRunLoop(MinecraftServer.java:9399) ~[intermediary-server.jar:?]
        at net.minecraft.server.MinecraftServer.method_29741(MinecraftServer.java:676) ~[intermediary-server.jar:?]
        at net.minecraft.server.MinecraftServer.method_29739(MinecraftServer.java:270) ~[intermediary-server.jar:?]
        at java.lang.Thread.run(Thread.java:831) [?:?]
```

**Comment by John-Paul-R** [comment]
Thanks for reporting. Am aware of the cause of this issue & am working on a fix.

**Comment by John-Paul-R** [comment]
This should be fixed as of [0.16.2](https://github.com/John-Paul-R/Essential-Commands/releases/tag/0.16.2-mc1.17.1)
